### PR TITLE
[RFC] [DX] Features cleanup

### PR DESCRIPTION
**Pull request by lchrusciel**
| Q               | A
| --------------- | ---
| Bug fix?        | yes
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | 
| License         | MIT

Little clean up, to follow up latest convention. Please share your ideas about naming of tags/concepts.

### CODE REVIEW

File: etc/behat/suites/ui/shipping/applying_shipping_fee.yml [hunk 1]
```diff
@@ -3,32 +3,29 @@
...
+                - sylius.behat.context.setup.zone
 
-                - sylius.behat.context.ui.checkout
                 - sylius.behat.context.ui.cart
-                - sylius.behat.context.ui.user
-
+                - sylius.behat.context.ui.checkout
             filters:
-                tags: "@cart && @ui"
+                tags: @applying_shipping_fee && @ui
```
Comment (pamil) [inline etc/behat/suites/ui/shipping/applying_shipping_fee.yml]: Missing quotations marks :water_buffalo:

File: etc/behat/suites.yml [hunk 2]
```diff
@@ -37,16 +37,19 @@ imports:
...
-    - suites/ui/cart.yml
-    - suites/ui/channel.yml
-    - suites/ui/checkout.yml
-    - suites/ui/locale.yml
-    - suites/ui/product.yml
-    - suites/ui/promotion.yml
+    - suites/ui/addressing/managing_countries.yml
+    - suites/ui/channel/multi_channeling.yml
+    - suites/ui/channel/theming.yml
+    - suites/ui/checkout/paying.yml
```
Comment (michalmarcinkowski) [inline etc/behat/suites.yml]: paying_for_order?

File: etc/behat/suites.yml [hunk 3]
```diff
@@ -37,16 +37,19 @@ imports:
...
 
-    - suites/ui/addressing.yml
-    - suites/ui/cart.yml
-    - suites/ui/channel.yml
-    - suites/ui/checkout.yml
-    - suites/ui/locale.yml
-    - suites/ui/product.yml
-    - suites/ui/promotion.yml
+    - suites/ui/addressing/managing_countries.yml
+    - suites/ui/channel/multi_channeling.yml
```
Comment (michalmarcinkowski) [inline etc/behat/suites.yml]: `supporting_multiple_channels`?

File: etc/behat/suites.yml [hunk 4]
```diff
@@ -37,16 +37,19 @@ imports:
...
+    - suites/ui/checkout/paying.yml
+    - suites/ui/locale/managing_locales.yml
+    - suites/ui/product/managing_products.yml
+    - suites/ui/promotion/applying_promotion_rules.yml
     - suites/ui/promotion/managing_promotions.yml
+    - suites/ui/promotion/receiving_discount.yml
+    - suites/ui/shipping/applying_shipping_fee.yml
     - suites/ui/shipping/managing_shipping_methods.yml
-    - suites/ui/taxation.yml
+    - suites/ui/taxation/applaying_taxes.yml
```
Comment (michalmarcinkowski) [inline etc/behat/suites.yml]: appl~~a~~ying

**Comment by lchrusciel** [comment]
@michalmarcinkowski review changes applied

**Comment by pjedrzejewski** [comment]
Looks good to me, but let's apply this after all pending PRs (for new admin) are merged and tweak everything at once. 👍

**Comment by michalmarcinkowski** [comment]
Good job Łukasz! 👍

**Pull request merged**